src/node_perf.cc

```diff
@@ -366,6 +366,25 @@ void Timerify(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(wrap);
 }
 
+// Notify a custom PerformanceEntry to observers
+void Notify(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Utf8Value type(env->isolate(), args[0]);
+  Local<Value> entry = args[1];
+  Context::Scope scope(env->context());
```

Comment:
I don’t think you strictly need this line. When the function is entered, the default Node.js context should already be on top of the Context stack.